os-server/src/main/java/app/openschool/course/module/quiz/api/dto/ModifyQuizDataRequest.java

```diff
@@ -0,0 +1,51 @@
+package app.openschool.course.module.quiz.api.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class ModifyQuizDataRequest {
+  @Schema(description = "Quiz title", example = "Quiz about ArrayList ")
+  private String title;
+
+  @Schema(
+      description = "Quiz description",
+      example = "With the ArrayList quiz you must answer at least 70% of the questions")
+  private String description;
+
+  @Schema(description = "Quiz maximum grade", example = "7")
+  private int maxGrade = -1;
```

Comment:
why these values are -1 ?

Reply:
I use update to check if that field is marked as a field subject to change in the request body or if the user does not want to change that field